.gitignore

```diff
@@ -163,6 +163,7 @@ cython_debug/
 # Private files
 
 config.py
+.DS_Store
```

Comment:
Every mac folder seems to get one of these (I think it's used for the os search indexing). These should get ignored.